src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/model/StartToolResult.groovy

```diff
@@ -0,0 +1,54 @@
+package de.tracetronic.jenkins.plugins.ecutestexecution.model
+
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted
+
+class StartToolResult implements Serializable {
+    private final String installationName
+    private final String toolExePath
+    private final String workSpaceDirPath
+    private final String settingsDirPath
+
+    StartToolResult(String installationName, String toolPath, String workSpaceDirPath, String settingsDirPath ){
+        this.installationName = installationName
+        this.toolExePath = toolPath
+        this.workSpaceDirPath = workSpaceDirPath
+        this.settingsDirPath = settingsDirPath
+    }
+
+    String getResult(){
+        return this.installationName != null ? "SUCCESS":"FAILED"
+    }
```

Comment:
Doesn't make sense to have a result, when we only create this result object in the success case